src/main/java/mesosphere/marathon/client/model/v2/UpgradeStrategy.java

```diff
@@ -0,0 +1,31 @@
+package mesosphere.marathon.client.model.v2;
+
+
+import mesosphere.marathon.client.utils.ModelUtils;
+
+public class UpgradeStrategy {
+
+    private double minimumHealthCapacity = 0.5;
```

Comment:
I am a little hesitant with these magic defaults. I am wondering how should we handle this at the client layer.
1. Maybe document this better (that reminds me that we need better documentation for our client), so that users understands what happens in the client layer.
2. Implement a default constructor that use defaults
3. Implement a two arg constructor that takes ```minimumHealthCapacity``` and ```maximumOverCapacity``` as parameters.

Thoughts ?